layouts/partials/hooks/body-end.html

```diff
@@ -2,3 +2,7 @@
   {{ partial "script.html" (dict "src" "js/tracing.js") -}}
 {{ end -}}
 {{ partial "script.html" (dict "src" "js/navScroll.js") -}}
+{{ $scrollToTopJs := resources.Get "js/scrollToTop.js" | minify | fingerprint }}
+<script src="{{ $scrollToTopJs.Permalink }}"></script>
+{{ $stickySearchJs := resources.Get "js/stickySearch.js" | minify | fingerprint }}
+<script src="{{ $stickySearchJs.Permalink }}"></script>
```

Comment:
```suggestion
{{ partial "script.html" (dict "src" "js/scrollToTop.js") -}}
{{ partial "script.html" (dict "src" "js/stickySearch.js") -}}
```

use the predefined template for adding a code snippet

Reply:
Alright mentor.

Reply:
Fixed as well! @svrnm .